scriptworker/test/test_gpg.py

```diff
@@ -590,19 +591,24 @@ def test_rebuild_gpg_home_signed(context, trusted_email, tmpdir):
 
 
 # get_git_revision, get_latest_tag {{{1
-# TODO these two tests should use a tarball with a test git repo inside
 @pytest.mark.asyncio
 async def test_get_git_revision():
-    parent_dir = os.path.dirname(__file__)
+    tar = tarfile.open(os.path.join(os.path.dirname(__file__), "data", "test_git_repo.tgz"))
+    tar.extractall(os.path.join(os.path.dirname(__file__), "data"))
+    parent_dir = os.path.join(os.path.dirname(__file__), "data", "testrepo")
     expected = subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=parent_dir)
```

Comment:
Thinking a bit about this, do you want to check against a hardcoded SHA (in this case it would be `3e9ed77c`) instead of getting the commit id from git?

On the other, using git itself to grab the sha ensures that the output of `sgpg.get_git_revision` is always the same format as what git returns.